code/game/machinery/computer/security.dm

```diff
@@ -369,15 +369,6 @@
 				P.name = "paper - 'Security Record'"
 				printing = 0
 
-		else if(href_list["print_p"])

```

Comment:
You should probably just comment this out, make a BYOND bug report, and put in a comment `//disabled due to byond bug <link here>`